packages/support/resources/views/components/section/index.blade.php

```diff
@@ -66,10 +67,8 @@
     {{
         $attributes->class([
             'fi-section',
-            match ($aside) {
-                true => 'fi-aside grid grid-cols-1 items-start gap-x-6 gap-y-4 md:grid-cols-3',
-                false => 'rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10',
-            },
+            'fi-aside grid grid-cols-1 items-start gap-x-6 gap-y-4 md:grid-cols-3' => $aside,
+            'rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10' => $contained,
```

Comment:
This doesn't seem right?
The previously `false` match arm only has a `$contained` condition here.